es-svc.yaml

```diff
@@ -10,7 +10,9 @@ spec:
   selector:
     component: elasticsearch
     role: client
+  clusterIP: 10.0.0.110
```

Comment:
For the Kibana component to talk to the elastic search service, it needed a URL( this is passed as an environment parameter to the docker environment of Kibana docker container). 

Without cluster IP won't be able to specify that URL of the elastic search.